### DISCUSSION TITLE
Reviewer Assignment Table

### DISCUSSION THREAD

**Discussion by TekaMesfinAbel** [MIT-Emerging-Talent/ET6-foundations-group-24#44] [Announcements]
# Reviewer Assignment Table

This table is for assigning two reviewers to each challenge solution. However, this doesn't mean that others cannot review your solution. The purpose of this table is to ensure that all group members are actively included in reviewing and collaborating on our work.

| **Challenge Creator**   | **Reviewer 1**         | **Reviewer 2**         |
|--------------------------|------------------------|-------------------------|
| Abel Mesfin Teka         | Viktoriya Haiduk    | Gai Samuel             |
| Gai Samuel               | Mohamed Elnageeb      | Abel Mesfin Teka        |
| Linah Khayri             | Viktoriya Haiduk      | Robel Mengsteab   |
| Khusro Sakhi             | Zeinab Mohammed       | Yonatan Yishkl Yifat     |
| Mohamed Elnageeb         | Khusro Sakhi              | Gai Samuel             |
| Robel Mengsteab          | Linah Khayri          | Khusro Sakhi           |
| Viktoriya Haiduk         | Mohamed Elnageeb      | Yonatan Yishkl Yifat   |
| Yonatan Yishkl Yifat     | Zeinab Mohammed       | Abel Mesfin Teka       |
| Zeinab Mohammed          | Linah Khayri    | Robel Mengsteab        |

### Additional Note
Challenge creators, please let your assigned reviewers know when your solution is ready for review.

**Comment by Khusro-S** [comment]
That was quick and nicely done @TekaMesfinAbel

**Comment by linahKhayri** [comment]
Well done @TekaMesfinAbel

**Comment by GaiSamuel** [comment]
Nice one!

**Comment by Zeinab15** [comment]
Thank you @TekaMesfinAbel you did great work ..

And also one of my issue is ready for review now @robiel0143 or @BanchiHub

**Comment by Mohamed-Elnageeb** [comment]
thanks abel

**Comment by ViktoriyaHaiduk** [comment]
Thanks Abel!
